tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "preserveWatchOutput": true,
     "skipLibCheck": true,
     "strict": true,
-    "target": "ES2015"
+    "target": "ES2015",
+    "verbatimModuleSyntax": true
```

Comment:
https://typescript-eslint.io/blog/consistent-type-imports-and-exports-why-and-how/#verbatim-module-syntax